RELEASE_NOTES.md

```diff
@@ -1,5 +1,46 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes.
 
+# Chef Client Release Notes 14.13:
+
+## Updated Resources
+
+### directory
+
+The `directory` has been updated to property set the `deny_rights` permsission on Windows.
+
+### service
+
+The `service` resource is now idempotent on SLES 11 systems.
+
+### cron
+
+The `cron` resource has been updated to advise users to use the specify properties rather than passsing values in as part of the `environment` property. This avoids a situation where a user could pass the differing values in both locations and receive unexpected results.
+
+### link
+
+The `link` resource includes improved logging on failure to help you debug what has failed.
```

Comment:
This could probably use some clarification. Did you mean..:
```suggestion
The `link` resource includes improved logging on failures to help you debug what has failed.
```
and/or
```suggestion
The `link` resource includes improved logging upon failure to help you debug what has failed.
```